src/core/dfn-panel.js

```diff
@@ -24,6 +24,11 @@ export async function run() {
   const panels = document.createDocumentFragment();
   for (const el of elems) {
     panels.append(createPanel(el));
+    // Make it possible to reach el by tabbing,
+    // allowing keyboard action as needed.
+    el.tabIndex = 0;
+    el.setAttribute("role", "link");
+    el.setAttribute("aria-haspopup", "dialog");
```

Comment:
I like having role=link - this resolves my 1 big concern that this was adding widget semantics to a structural role. Links are widgets so it is natural to focus them. 
1 improvement would be to add an accessible description to indicate that this will "show usages of foo" - the easiest way to do this would be using the title attribute.